rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/jvminterop/JavaSystemInterface.kt

```diff
@@ -94,7 +114,28 @@ open class JavaSystemInterface(
     }
 
     override fun findFunction(globalSymbolTable: ISymbolTable, name: String): Function? {
-        TODO("not implemented") // To change body of created functions use File | Settings | File Templates.
+        return functions.computeIfAbsent(name) {
```

Comment:
Is there a test unit about this feature?

Reply:
Not yet

Reply:
We will postpone it